docs/sphinx/user/inputs_TKE_Sources.rst

```diff
@@ -0,0 +1,11 @@
+.. _inputs_tke_sources:
+
+Section: TKE Sources
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+   
+.. input_param:: TKE.source_terms
+
+   **type:** String(s), optional
+   
+These terms are used when the turbulence model includes a transport equation for  turbulent 
```

Comment:
is there a check in the code enforcing this?

Reply:
I check in the source term if the RANS model is KLAxell but there is no check in the RANS model if we have enabled the source term.

Reply:
If the activation of that source term is needed, let's add a check for it and/or activate it automatically.